install_sct

```diff
@@ -625,46 +625,14 @@ python -m pip install -U "pip!=21.2.*"
 
 ## Install the spinalcordtoolbox into the Conda venv
 print info "Installing Python dependencies..."
-# Check if a frozen version of the requirements exist (for release only)
-if [[ -f "requirements-freeze.txt" ]]; then
-  print info "Using requirements-freeze.txt (release installation)"
-  REQUIREMENTS_FILE="requirements-freeze.txt"
-else
-  # Not a package
-  print info "Using requirements.txt (git installation)"
-  REQUIREMENTS_FILE="requirements.txt"
-fi
 (
   # We use "--ignore-installed" to preserve the version of `certifi` installed into the conda
   # env, which prevents https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3609
-  pip install -r "$REQUIREMENTS_FILE" --ignore-installed certifi &&
   print info "Installing spinalcordtoolbox..." &&
-  pip install -e .
+  pip install -e . --ignore-installed certifi -f https://download.pytorch.org/whl/cpu/torch_stable.html
 ) ||
   die "Failed running pip install: $?"
 
-## Check if CPU supports AVX instruction set, and install precompiled TensorFlow if it isn't,
```

Comment:
not sure if we should remove

Reply:
This will need to be updated (rather than removed). Though, I'm fairly certain that we'll be able to find (or compile ourselves) new `.whl` files for whichever version of TF we upgrade, though. There are lots of people out there who have an interest in providing support for AVX-less CPUs. :)

For now, you can just ignore this? I think it'll be easiest to just focus on getting TF working for our CI runners. And then, once we can confirm that it's possible to upgrade, we can go back and fill in the gaps for AVX-less users.